libraries/ui-library/src/components/six-header/test/six-header.spec.tsx

```diff
@@ -2,7 +2,7 @@ import { newSpecPage } from '@stencil/core/testing';
 import { SixHeader } from '../six-header';
 
 describe('six-header', () => {
-  it('renders', async () => {
+  xit('renders', async () => {
```

Comment:
Same here why is this test muted? I'm not a fan of bringing dead code into the main branch. Is the test currently broken?